### ISSUE TITLE
New Feature - Marking Dice

### ISSUE THREAD

**Issue by b01001011** [funprog/funbot#15]
We can mark anything on each side of the dice.

For example,
- If the request is `/dice 2d{@junoya77, @masunghoon, @taewanme, @younggi}`,
- then the response would be `@taewanme, @junoya77`

**Comment by jwChung** [comment]
좋은 생각이네요.

그런데 괄호 안에 슬랙 아이디일 필요가 있나요?  `{이경준, 정진욱}` 이런 사용도 가능한가요?

**Comment by b01001011** [comment]
Good call! Why don't we make it to take any types of input? I think even this would work, `{1, 2, 3, jungsu, 4, 6, 6}`.

**Comment by jwChung** [comment]
@kwoolytech 걍 `String` 타입이라도 충분할까요?

@funprog/core 이 이슈 채택해도 되겠습니까?

**Comment by younggi** [comment]
아이디나 이름으로 뽑기하는 기능 추가 찬성합니다.

**Comment by naxxster** [comment]
어떤 단위로 구분할 지에 대해서만 명확해진다면 유용할 것 같네요.

**Comment by jwChung** [comment]
@younggi, @naxxster 님들이 찬성 의견 주셔서, 이슈 채택의 의미로 enhancement 라벨 부여하였습니다.

**Comment by b01001011** [comment]
@jwChung Yup, `String` sounds good to me. ty all!

**Comment by myeesan** [comment]
중복을 허용하나요? 예를 들어 `2d{1, 2, 3, 4}` 인 경우,  `1, 1` 결과 값이 나올 수 있나요?

**Comment by jwChung** [comment]
@myeesan  먼저 #6 을 구현한 후 고민하는 것이 어떨까요? #6 이 기존의 중복을 허용하는 #5 기능을 덮어 쓸지 아니면 #5 은 그대로 두고 추가적으로 구현해야할지 결정할필요가 있겠습니다. 이 결정이 먼저 내려진 후에 고려되는 것이 좋을 것 같습니다.

**Comment by myeesan** [comment]
@jwChung 넵!
